UpskillWikipediaUITests/PageObjects/ArticlePage.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class ArticlePage: BaseAppPage {
+    
+    // MARK: - Elements
+    private lazy var webViewText = app.webViews.staticTexts
+    private lazy var navigationBar = app.navigationBars.firstMatch
+    private lazy var backButton = app.buttons["Back"].firstMatch
+    
+    // MARK: Actions
+    func tapBackButton() {
+        _ = backButton.waitForExistence(timeout: TestTimeouts.generalUiElementTimeout)
+        backButton.tap()
+    }
+    
+    // MARK: - Helpers
+    func isTextOnPageDispplayed(text: String) -> Bool {
+        _ = webViewText.element.waitForExistence(timeout: TestTimeouts.generalUiElementTimeout)
+        return webViewText.element(withLabel: text).firstMatch.exists
```

Comment:
You could make it a bit simpler - elements don't have to be declared as `lazy var` - they can be a functions with parameters too:
```
private func staticTextInsideArticleView(expectedText: String) -> XCUIElement {
    return app.webViews.staticTexts.element(withLabel: expectedText).firstMatch
}
```
And later you can use it in method `isTextOnPageDispplayed()` like this:
```
func isTextOnPageDispplayed(text: String) -> Bool {
    return staticTextInsideArticleView(text).waitForExistence(timeout: TestTimeouts.generalUiElementTimeout)
}